packages/docusaurus-theme-translations/locales/ru/theme-common.json

```diff
@@ -4,6 +4,7 @@
   "theme.CodeBlock.copied": "Скопировано",
   "theme.CodeBlock.copy": "Скопировать",
   "theme.CodeBlock.copyButtonAriaLabel": "Скопировать в буфер обмена",
+  "theme.CodeBlock.wordWrapToggle": "Toggle word wrap",
```

Comment:
@slorber @lex111 Not a blocker but I suggest we help maintain `ru`/`fr`/`zh` fully translated? This way at least we have some locales that are always working.

Reply:
Yep, good idea, I often forget about it.